php/class-photonfill.php

```diff
@@ -400,6 +405,31 @@ public function add_width_for_captions( $html, $id, $caption, $title, $align, $u
 			return $html;
 		}
 
+		/**
+		 * Ajax responder to get image object.
+		 *
+		 */
+		function ajax_get_img_object() {
+			check_ajax_referer( 'photonfill_get_img_object', 'nonce' );
+			if ( ! empty( $_POST['attachment'] ) ) {
+				$attachment_id = absint( $_POST['attachment'] );
+				echo $this->get_attachment_image( $attachment_id, 'full', array( 'style' => 'width:100%' ) );
+			}
+			exit();
+		}
+
+		/**
+		 * Fix issues with fieldmanager loading images
+		 *
+		 */
+		function set_fieldmanager_media( $preview, $value, $attachment ) {
+			if ( ! empty( $attachment->ID ) && strpos( $attachment->post_mime_type, 'image/' ) === 0 ) {
+				$preview = esc_html__( 'Uploaded image:', 'fieldmanager' ) . '<br />';
```

Comment:
the i18n domain cannot be "fieldmanager". it should be "photonfill" or whatever this one is